schema/site_metadata.json

```diff
@@ -39,15 +48,211 @@
         },
         "long": {
-            "description": "Longitude of the site in WGS84 coordinates, as indicated by a map marker",
-            "type": "number"
+            "description": "Longitude of the site in WGS84 coordinates, East of Greenwich, as indicated by a map marker",
+            "type": "number",
+            "examples": [12.4922]
+        }
+      }
+    },
+    "links": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "docs": {
+          "description": "Folder where the project documentation is stored",
+          "type": "string",
+          "examples": ["file://home/user/docs", "https://drive.google.com/corp/drive/folders/iamadrivelocation"]
+        },
+        "folder": {
+          "description": "Folder where the UDMI site model is stored",
+          "type": "string",
+          "examples": ["file://home/user/udmi_site_model", "https://drive.google.com/corp/drive/folders/iamadrivelocation"]
+        },
+        "image": {
+          "description": "Image of the site",
+          "type": "string",
+          "examples": ["file://home/user/image.png", "https://drive.google.com/corp/drive/folders/iamadrivelocation"]
+        },
+        "repo": {
+          "description": "Source repository where the UDMI site model is stored",
+          "type": "string",
+          "examples": ["https://github.com/faucetsdn/udmi_site_model", "git@github.com:faucetsdn/udmi_site_model.git"]
+        }
+      }
+    },
+    "counts": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "modeled": {
+          "description": "Count of modeled devices",
+          "type": "number",
+          "examples": [42]
+        },
+        "validated": {
+          "description": "Count of validated devices",
+          "type": "number",
+          "examples": [42]
+        },
+        "registered": {
+          "description": "Count of registered devices",
+          "type": "number",
+          "examples": [42]
+        },
+        "discovered": {
+          "description": "Count of discovered devices",
+          "type": "number",
+          "examples": [42]
+        }
+      }
+    },
+    "parameters": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "area": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "net_occupied_area": {
+              "type": "object",
+              "additionalProperties": false,
+              "properties": {
+                "value": {
+                  "description": "Net Occupied Area of the site",
+                  "type": "number",
+                  "examples": [1234, 256, 997765]
+                },
+                "unit": {
+                  "description": "Unit used for carbon the Net Occupied Area",
+                  "type": "string",
+                  "examples": ["m2", "sqf"]
```

Comment:
NB: Not sure if we want to be more picky about this... e.g. an enum?  I'm just worried about the creep where people to something like "f^2" as a unit rather than "sqf" -- or we just need to plan on some other validation step (if/when) this becomes an issue.  Also possible (I think) to impose a regex on this like "(m2|sqf)" which will at least limit the initial options without the pain of an enum.